ci-operator/step-registry/ipi/conf/aws/byo-ipv4-pool-public/ipi-conf-aws-byo-ipv4-pool-public-commands.sh

```diff
@@ -92,4 +92,50 @@ cat ${CONFIG_PATCH}
 yq-go m -x -i "${CONFIG}" "${CONFIG_PATCH}"
 
 # save pool id for post-check
-echo ${pool_id} > ${SHARED_DIR}/ipv4_pool_id
\ No newline at end of file
+echo "${pool_id}" > ${SHARED_DIR}/ipv4_pool_id
+
+if [[ "${USE_PUBLIC_IPV4_POOL_INGRESS-}" != "yes" ]]; then
+  echo "USE_PUBLIC_IPV4_POOL_INGRESS(${USE_PUBLIC_IPV4_POOL_INGRESS-}) is not enabled, skipping Ingress configuration"
+  exit 0
+fi
+
+echo "Creating Elastic IPs for each zone..."
+# Create a new Elastic IP for each zones
+for i in $(seq 0 $((zone_count - 1))); do
+  aws ec2 allocate-address \
+    --domain vpc \
+    --region "${AWS_REGION}" \
+    --public-ipv4-pool "${pool_id}" > /tmp/eip-"${i}".json
```

Comment:
TODO create tags to resource.

The following tags must be created:
- `Name: ci-op-<random>-eip-lb-ingress` <-- this may follow the CAPA standard which has the standard suffix `eip-<role-resource>`
- `kubernetes.io/cluster/<clusterName or ID (if available in artifacts)>: shared`
- `expirationDate: <timestamp>`




Examples of `<role-resource>` on CAPA:
- lb-apiserver
- common
- ec2-i-instanceId